eemeter/caltrack.py

```diff
@@ -1214,14 +1229,26 @@ def caltrack_method(
         else:
             num_parameters = 0
 
-        predicted = _caltrack_predict_design_matrix(
+        predicted_avgs = _caltrack_predict_design_matrix(
             best_candidate.model_type,
             best_candidate.model_params,
             data,
             input_averages = True,
             output_averages = True,
         )
-        model_result.metrics = ModelMetrics(data.meter_value, predicted, num_parameters)
+        model_result.avgs_metrics = ModelMetrics(data.meter_value, predicted_avgs, num_parameters)
+
+        predicted_totals = _caltrack_predict_design_matrix(
+            best_candidate.model_type,
+            best_candidate.model_params,
+            data,
+            input_averages = True,
```

Comment:
This proved useful very quickly! My initial skepticism is now entirely vanquished.